src/index.html

```diff
@@ -17,3 +17,32 @@ <h1 class="logo">MA</h1>
     <script src="scripts/main.js"></script>
   </body>
 </html>
+<script>
+  const logo = document.querySelector('.logo');
+
+  const promise1 = new Promise((resolve) => {
+    logo.addEventListener('click', () => {
+      resolve('Promise was resolved!');
+    });
+  });
+
+  const promise2 = new Promise((_, reject) => {
+    setTimeout(() => {
+      reject('Promise was rejected!');
+    }, 3000);
```

Comment:
The `promise2` is rejected after 3 seconds regardless of any conditions. This may not be the desired behavior in a real-world scenario. Consider under what conditions you would want to reject the promise and handle it accordingly.